e2e/betweenness_centrality_test/test_cugraph_large_directed/input.cyp

```diff
@@ -0,0 +1,59 @@
+MERGE (a:Node {id: 0});
+MERGE (a:Node {id: 1});
+MERGE (a:Node {id: 2});
+MERGE (a:Node {id: 3});
+MERGE (a:Node {id: 4});
+MERGE (a:Node {id: 5});
+MERGE (a:Node {id: 6});
+MERGE (a:Node {id: 7});
+MERGE (a:Node {id: 8});
+MERGE (a:Node {id: 9});
+MERGE (a:Node {id: 10});
+MERGE (a:Node {id: 11});
+MERGE (a:Node {id: 12});
+MERGE (a:Node {id: 13});
+MERGE (a:Node {id: 14});
```

Comment:
I don't see the need for this when you have `MERGE` for nodes in create RELATION.

Reply:
Also it is not consistent with other tests.

Reply:
We have to because there are some nodes which are isolated, and here they are created.